tool/tsh/tsh.go

```diff
@@ -268,6 +272,7 @@ func Run(args []string, underTest bool) {
 		identityfile.FormatOpenSSH)).Default(string(identityfile.DefaultFormat)).StringVar((*string)(&cf.IdentityFormat))
 	login.Flag("request-roles", "Request one or more extra roles").StringVar(&cf.DesiredRoles)
 	login.Arg("cluster", clusterHelp).StringVar(&cf.SiteName)
+	login.Flag("disable-browser", noBrowserHelp).BoolVar(&cf.NoBrowser)
```

Comment:
I think we should add this feature but change the CLI experience.

I looked around, and I would suggest that we add a `--open` flag, this can then be extended in the future to add  `--browserTarget=browserTarget`


The new Flag will be `--open=false`

### TSH Login without auto-open of Browser.  

```
tsh login --proxy=teleport.example.com --open=false 
```

Reply:
How about make it `--browser=none` to not open a browser at all? That way we can extend this in future to make use of things like `--browser=chrome` etc.

Reply:
yeah, that's a very elegant solution. I've updated the question to suggest it.